packages/wdio-browserstack-service/src/exitHandler.ts

```diff
@@ -0,0 +1,33 @@
+import { spawn } from 'node:child_process'
+import path from 'node:path'
+import BrowserStackConfig from './config.js'
+import FunnelTestEvent from './instrumentation/funnelInstrumentation.js'
+import { fileURLToPath } from 'node:url'
+
+const __filename = fileURLToPath(import.meta.url)
+const __dirname = path.dirname(__filename)
+
+export function setupExitHandlers() {
+    process.on('exit', (code) => {
+        const args = shouldCallCleanup(BrowserStackConfig.getInstance())
+        if (Array.isArray(args) && args.length) {
+            const childProcess = spawn('node', [`${path.join(__dirname, 'cleanup.js')}`, ...args], { detached: true, stdio: 'inherit', env: { ...process.env } })
+            childProcess.unref()
+            process.exit(code)
+        }
+    })
+}
+
+export function shouldCallCleanup(config: BrowserStackConfig): string[] {
+    const args: string[] = []
+    if (!!process.env.BS_TESTOPS_JWT && !config.testObservability.buildStopped) {
```

Comment:
I am seeing a bunch of these environment variables here. Is there a place where these are documented?

Reply:
`BS_TESTOPS_JWT` -> We use this env variable for internal purposes only.

When we launch the session in parent process we receive JWT token, in order to authenticate next requests. The next requests are sent from worker processes, so in order to pass this JWT token from parent to workers we are using the env variables. The other env variables `BS_TESTOPS_ALLOW_SCREENSHOTS` & `BS_TESTOPS_BUILD_COMPLETED` are also used for similar purposes.

Also none of this env variables are added in the PR, they are already existent in the code, I am just using them. As these are internal env variables, we didn't document them

Reply:
It would be helpful to have them defined in a e.g. `constants.ts` file where you export their value with a comment for folks who don't know these environment variables. This would make the code easier to maintain. Remember, that currently maybe all your engineers know their meaning but the future generation of engineers need to know this too.

Reply:
Yes, we should do that.  Have changed the code to define the env string in constants, and using it everywhere https://github.com/webdriverio/webdriverio/pull/12392/commits/e82ce796bf4a0539c0293c0ec146c5684dd404f2